### DISCUSSION TITLE
wonderful library but, I'm struggling with it - big time

### DISCUSSION THREAD

**Discussion by ihajat** [patrykandpatrick/vico#837] [Questions]
### Question

1) I changed the color of the text of the marker, sometimes I see the color change, but most times it's the same color as the line color? is that a bug? 2) how do I add extra lines of text in the label? 3) how do I find the index of the array that was clicked so that I can change the text of the label?

### Vico version(s)

latest

### UI framework(s)

Jetpack Compose

**Comment by Gowsky** [comment]
Hi @ihajat,
1. Answered in #834.
2. See the `lineCount` parameter.
3. What do you mean by “array”?
